### ISSUE TITLE
BOLT English Propbank Release Data

### ISSUE THREAD

**Issue by celine-lee** [propbank/propbank-release#10]
The 'BOLT English PropBank and Sense -- Discussion Forum, SMS/Chat, and Conversational Telephone Search' was released a few weeks ago as LDC2020T21. 

I am wondering if the annotations and conversion files will be available here?

**Comment by timjogorman** [comment]
Yes!  Sorry for the delay -- I'm hoping to get those posted this weekend.

**Comment by celine-lee** [comment]
Great! Thank you so much.

**Comment by celine-lee** [comment]
Hi! Sorry to prod, but are there any updates on the timeline for this? Thanks!

**Comment by MarthaSPalmer** [comment]
LDC just released it at then end of September.

Martha


On Oct 8, 2020, at 7:19 PM, Celine Lee <notifications@github.com> wrote:

﻿

Hi! Sorry to prod, but are there any updates on the timeline for this? Thanks!

**Comment by timjogorman** [comment]
Unfortunately, to my immense frustration, LDC decided to remove the source text from the Propbank release we delivered, and so we can only set up access to datasets where LDC has released that source text in other releases. 

I've put up a branch ("addbolt') which hopefully gives access the largest part of the BOLT release -- the discussion forum subset -- but it requires two LDC packages for the source trees, LDC2020T09 and LDC2019T15.  (It's missing 10 files that are causing problems, out of the 852 total -- I'll push to master when those are sorted out).  

As for the SMS and CTS corpora -- unless we can convince  LDC to update to update this to include the source text, they will need to wait for BOLT SMS and CTS treebank releases.
